src/azul/transformer.py

```diff
@@ -188,7 +192,7 @@ def translate_fields(cls,
                         raise KeyError(f'Key {key} not defined in field_types')
                     except TypeError:
                         raise TypeError(f'Key {key} not defined in field_types')
-                    new_dict[key] = cls.translate_fields(val, field_type, forward=forward)
+                    new_dict[key] = cls.translate_fields(val, field_type, forward=forward, path=path + [key])
```

Comment:
This is the copy.

```suggestion
                    new_dict[key] = cls.translate_fields(val, field_type, forward=forward, path=(**path, key))
```

should be more efficient.